bindings/pyroot/pythonizations/python/ROOT/_pythonization/_rdfdescription.py

```diff
@@ -0,0 +1,19 @@
+# Author: Ivan Kabadzhov, Vincenzo Eduardo Padulano CERN  01/2022
+
+################################################################################
+# Copyright (C) 1995-2022, Rene Brun and Fons Rademakers.                      #
+# All rights reserved.                                                         #
+#                                                                              #
+# For the licensing terms see $ROOTSYS/LICENSE.                                #
+# For the list of contributors see $ROOTSYS/README/CREDITS.                    #
+################################################################################
+
+from . import pythonization
+
+@pythonization("RDFDescription", ns="ROOT::RDF")
+def pythonize_rdataframe(klass):
```

Comment:
Congrats for your first pythonization!

I would call this `pythonize_rdfdescription` to be consistent in the naming.

Reply:
corrected in the latest commit (and also the other typo)